Multisig/UI/App/UpdateAppViewController.swift

```diff
@@ -17,11 +17,11 @@ class UpdateAppViewController: UIViewController {
         var description: String {
             switch self {
             case .required:
-                return "Your version of the Safe app is not supported anymore since it is too old. Please update your app"
+                return "Your version of the Safe{Wallet} app is not supported anymore since it is too old. Please update your app"
```

Comment:
may be "Safe {Wallet} mobile app for distinguishing between web app and mobile app".?

Reply:
It should be clear from the context that app itself is not supported.